annlite/container.py

```diff
@@ -21,20 +21,20 @@
 class CellContainer:
     def __init__(
         self,
-        dim: int,
+        n_dim: int,
```

Comment:
please, can we avoid this being a breaking change as well? A lot of users are using ANNLite and it would be cool if we can avoid annoying them

Reply:
no, the parameter `dim` will also work without annoying the existing users.